.github/ISSUE_TEMPLATE/general.md

```diff
@@ -0,0 +1,43 @@
+---
+name: General Feedback
+about: Any general feedback (neither feature request nor bug reports)
+---
+
+## Prerequisites
+
+- First, many thanks for taking part in the community. We really appreciate that.
+- Read the [contributing guidelines](http://solidity.readthedocs.io/en/latest/contributing.html).
+- Support questions are better asked in one of the following locations:
+	- [Solidity chat](https://gitter.im/ethereum/solidity)
+	- [Stack Overflow](https://ethereum.stackexchange.com/)
+- Ensure the issue isn't already reported.
+
+*Delete the above section and the instructions in the sections below before submitting*
+
+## Description
+
+If this is a feature request, explain why it should be added. Specific use-cases are best.
+
+For bug reports, please provide as much *relevant* info as possible.
+
+### Test Source
```

Comment:
If it is not a bug/feature why to include these headings?

Reply:
this was apparently a copy'n'paste bug, I didn't remove it here (yet).